### ISSUE TITLE
RCS1132 false positive with `.ToString()` in records

### ISSUE THREAD

**Issue by eduherminio** [JosefPihrt/Roslynator#1012]
**Product and Version Used:**

Roslynator 2022 4.2.0 (VS 2022 extension)
.NET 6

**Steps to Reproduce**:

```csharp
Console.WriteLine(new IntPointWithValue<string>("id", 0, 1).ToString());

public record IntPoint(int X, int Y)
{
    public override string ToString()
    {
        return $"[{X}, {Y}]";
    }
}

public record IntPointWithValue<T>(T value, int X, int Y) : IntPoint(X, Y)
{
    // Comment and uncomment
    // public override string ToString() => base.ToString();    // RCS1132, but without this, it's not inheriting base behavior
}
```

**Actual Behavior**:
` RCS1132 Remove redundant overriding member`

**Expected Behavior**:
No RCS1132, since we're indeed changing the behavior with that override

**Comment by eduherminio** [comment]
(Closely?) related to https://github.com/JosefPihrt/Roslynator/issues/744.
